addons/account/views/account_move_views.xml

```diff
@@ -96,36 +96,34 @@
                     <field name="company_currency_id"/>
                     <field name="partner_id"/>
```

Comment:
Not necessary, already defined in the template.

You can remove all fields (in all archs) defined above the template, if the field is also used (with the `<field>` tag) in the template, or if the field isn't used at all in the template (i.e. not used in a t-if expression, for instance).